app/services/get-tab-title.js

```diff
@@ -1,21 +1,24 @@
 const organisationUnitConstants = require('../constants/organisation-unit')
 
-module.exports = function (pageTitle, subNav, organisationLevel) {
-  let selectedSubNavTitle = ''
+module.exports = function (pageTitle, pageSubTitle, subNav, organisationLevel) {
+  let selectedSubNavTitle = null
   subNav?.forEach(function (item) {
     if (item.active) {
       selectedSubNavTitle = item.title
     }
   })
+  if (selectedSubNavTitle === 'OMIC Overview') {
```

Comment:
i don't like this but it's probably the easiest option. That is unless we can convince Rose it's inconsistent and change the H1 (or change what's shown in the tab title)